### PR TITLE
Home Page with Quiz Categories

### PR DESCRIPTION
**Pull request by samflab**
Here's a home page that displays the quiz categories, a search bar to search for the quiz, and a button to create new quizzes.
`Note: The site is not responsive yet`
Deployed at: https://deploy-preview-1--xquizx.netlify.app/
## Screenshots:
image

### CODE REVIEW

File: css/header.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.header{
+    border-bottom: none !important;
+    box-shadow: rgba(78, 86, 88, 0.1) 0px 10px 15px -3px, rgba(186, 223, 238, 0.05) 0px 4px 6px -2px !important;
```
Comment (rutvikpumak) [inline css/header.css]: why important here

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+.search-inp {
+  padding: 0.5rem 1rem;
+  border-radius: 50px;
+  border: 1px solid gray;
+  outline: none;
+  position: relative;
+}
+
+.fl-btn {
+    background-color: var(--primary-light)  !important;
```
Comment (rutvikpumak) [inline css/index.css:33]: again you have used imp

File: css/header.css (hunk 1 above)
Comment (samflab) [reply css/header.css:3]: I have used the components from my component library. I wanted to override the existing CSS. Without `!important` it wasn't overriding the styles. So, I could either use inline style or use `!important` keyword.

File: css/index.css (hunk 2 above)
Comment (samflab) [inline css/index.css]: same reason as above

File: css/header.css (hunk 1 above)
Comment (samflab) [reply css/header.css]: fixed the issue.

**Review by rutvikpumak — APPROVED** [review]

**Review by SK02K1 — APPROVED** [review]

**Pull request merged**